_search-plugins/star-tree-index.md

```diff
@@ -39,9 +39,9 @@ The following image illustrates a standard star-tree index structure.
 
 <img src="{{site.url}}{{site.baseurl}}/images/star-tree-index.png" alt="A star-tree index containing two dimensions and two metrics" width="700">
 
-Sorted and aggregated star-tree documents are backed by `doc_values` in an index. `doc_values` use the following pattern:
+Sorted and aggregated STIX documents are backed by `doc_values` in an index. `doc_values` use the following pattern:
```

Comment:
Sorted and aggregated STIX documents are backed by `doc_values` in an index. The columnar data part of `doc_values` are stored with following properties :